lib/cookie/secureContext.ts

```diff
@@ -0,0 +1,140 @@
+import { URL } from 'url';
+import { isIP, isIPv4, isIPv6 } from 'net';
+
+/**
+ * Checks if the given IPv4 address is in the 127.0.0.0/8 loopback range.
+ *
+ * @remarks
+ * @param address - The IPv4 address to check.
+ * @returns `true` if the address is loopback, otherwise `false`.
+ */
```

Comment:
```suggestion
```

Documentation is only enforced for functions exposed in the public API. This code is simple enough to be self-describing so the doc comment seems unnecessary.